bucket/kops.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "1.11.0",
+    "description": "Kubernetes Operations (kops) - Production Grade K8s Installation, Upgrades, and Management.",
+    "homepage": "https://github.com/kubernetes/kops/",
+    "license": "Apache-2.0",
```

https://github.com/kubernetes/kops#prerequisite

> kubectl is required

```suggestion
    "license": "Apache-2.0",
    "depends": "kubectl",
```